frontend/public/components/monitoring/dashboards/index.tsx

```diff
@@ -690,6 +742,22 @@ const MonitoringDashboardsPage: React.FC<MonitoringDashboardsPageProps> = ({ mat
         return acc;
       }, []);
 
+  const handleDashboardChange = (val: string) => {
+    let url = `/monitoring/dashboards/${val}`;
+    const params = new URLSearchParams('?');
+    const refreshInterval = getQueryArgument('refreshInterval');
+
+    // Time range options are reset to their defaults and the refresh Interval is being persisted
+    if (refreshInterval) {
+      params.set('refreshInterval', refreshInterval);
+    }
```

Comment:
You could do `const params = new URLSearchParams({ refreshInterval: getQueryArgument('refreshInterval') })`